lib/wkt.js

```diff
@@ -111,7 +111,11 @@ function cleanWKT(wkt) {
       wkt.units = 'meter';
     }
     if (wkt.UNIT.convert) {
-      wkt.to_meter = parseFloat(wkt.UNIT.convert, 10);
+      if (wkt.type === 'GEOGCS') {
+        wkt.to_meter = parseFloat(wkt.UNIT.convert, 10)*6370997;
```

Comment:
You shouldn't be using the constant `6370997` directly here. This value has to be taken from the correct datum.